src/Internal/DependencyManagement/DBServiceProvider.php

```diff
@@ -90,6 +91,7 @@ public function register() {
 
 		// Share DB migrations
 		$this->share_migration( MigrationVersion141::class, MerchantIssueTable::class );
+		$this->share_migration( Migration20211228T1640692399::class, ShippingRateTable::class );
```

Comment:
Is this naming temporary until we have the know the version this will be implemented?

Reply:
Not really. I previously used the version number in the class name but that isn't very scalable. We usually don't know the version that this migration is going to be released on and also, for example, if two people are going to be committing a migration class for the same version we would get a conflict. I decided to use a datetime+unix-timestamp naming for them to resolve this.

The migration is still tied to a version `x.x.x`, which will get replaced by woorelease.

Reply:
Ok that makes sense. Although if we are going to have multiple migration classes for a version it would probably be better to merge them.